kayenta-judge/src/main/scala/com/netflix/kayenta/judge/classifiers/metric/BaseMetricClassifier.scala

```diff
@@ -41,6 +41,20 @@ object MetricDirection{
   }
 }
 
+sealed trait NanStrategy
```

Comment:
For consistency with other projects (Apache Math) this should be defined as `NaNStrategy`

Reply:
Will fix.  I also added a test (gasp!)